DCA/adapter.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
```

Comment:
Let's have nice sorting of imports (first -- external libs import, then external from, then local imports) and separation lines between. See example https://github.com/prime-slam/line-detection-association-dockers/blob/main/detectors/F-Clip/adapter.py

Reply:
I don't see any fixes